### PR TITLE
[3.3] Scaffolders documentation; Change the name of the VSC extension. 

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: getting-started/vs-code-integration/scaffolders.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+page_title: Telerik UI for Blazor Code Scaffolders - Visual Studio Code Integration
+description: Learn how to add a new component page in Visual Studio Code via code scaffolder templates.
+slug: getting-started-vs-code-integration-scaffolders
+position: 20
+---
+
+
+# Scaffolders
+
+The Telerik Visual Studio Code extension provides a handy feature for increased developer productivity – scaffolders. They allow you to quickly add component pages to your application and setup the parameters of the component through an UI.
```
Comment (dimodi) [inline getting-started/vs-code-integration/scaffolders.md:12]: I would suggest to reduce the marketing-sounding tone in docs :)

```suggestion
The Telerik Visual Studio Code extension provides scaffolders for increased developer productivity. They allow you to quickly add component pages to your application and setup the parameters of the component through an UI.
```

File: getting-started/vs-code-integration/scaffolders.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Get the Extension
+
+To use the **Telerik UI for Blazor Scaffolders**, install the Telerik UI for Blazor Visual Studio Code Extension. @[template](/_contentTemplates/common/general-info.md#vs-code-x-download)
+
+## Add a new component page
+
+1. Right-click on the name of the solution
+1. From the popup menu select the ``
```
Comment (dimodi) [inline getting-started/vs-code-integration/scaffolders.md:21]: ?

File: getting-started/vs-code-integration/scaffolders.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+1. From the UI form, select the desired component and set the available parameters
+
+>caption Add a new component page
+
+## Available Scaffolded Components
+
+| Component |
```
Comment (dimodi) [inline getting-started/vs-code-integration/scaffolders.md:28]: Why not sort the components alphabetically?

**Review by dimodi — CHANGES_REQUESTED** [review]
Add missing word in steps-by-step instructions

File: getting-started/vs-code-integration/scaffolders.md (hunk 3 above)
Comment (svdimitr) [reply getting-started/vs-code-integration/scaffolders.md:29]: Did it by component "popularity", started with the grid and everything else is listed below.

**Review by jivanova — APPROVED** [review]

**Pull request merged**